demo/src/features/mesh/extract.rs

```diff
@@ -122,7 +123,20 @@ impl ExtractJob for MeshExtractJob {
             })
             .collect();
 
+        let static_resources = extract_context
+            .render_resources
+            .fetch::<MeshStaticResources>();
+
+        let asset_manager = extract_context
+            .render_resources
+            .fetch::<AssetManagerRenderResource>();
+
+        let depth_material = asset_manager
+            .get_material_pass_by_name(&static_resources.depth_material, "Depth")
```

Comment:
I was torn on that. I felt that the index lookup might be more brittle (e.g. if we add a multi-view depth pass for point lights) but obviously right now it doesn't matter since there's only the one pass.

Reply:
We could consider a function like `get_only_material_pass` (or some other better name!) that asserts there is exactly 1 material pass in the material and returns it. That would make intent clear and avoid silent failure of using the wrong material pass.